src/main/java/com/synopsys/integration/detect/tool/UniversalToolsResultBuilder.java

```diff
@@ -31,10 +35,11 @@ public void addDetectorToolResult(DetectorToolResult detectorToolResult) {
         detectCodeLocations.addAll(detectorToolResult.getBomToolCodeLocations());
         detectToolGitInfo = detectorToolResult.getGitInfo();
         anyFailed = detectorToolResult.anyDetectorsFailed() || anyFailed;
+        applicableDetectorTypes.addAll(detectorToolResult.getApplicableDetectorTypes());
```

Comment:
Copies the elements over, does not reference the same set

Reply:
The above is no longer applicable after some review comments prompted a change. See latest commit:b275dd59cff970d2a0b0f805a2af41360173f515